README.md

```
@@ -1,32 +1,61 @@
 
 ![npm version](https://img.shields.io/npm/v/dev-stamp) ![MIT License](https://img.shields.io/badge/license-MIT-blue.svg)
 
 ---
 
+## 📜 Table of Contents
+
+- [🚀 What is Dev Stamp?](#-what-is-dev-stamp)
+- [🎯 Use Cases](#-use-cases)
+- [🔥 Why You'll Love It](#-why-youll-love-it)
+- [📦 Installation](#-installation)
+- [✍️ Quick Example](#-quick-example)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)